source-hubspot-native/source_hubspot_native/api.py

```diff
@@ -349,7 +349,7 @@ async def fetch_changes(
 
     recent.sort()  # Oldest updates first.
 
-    for batch_it in itertools.batched(recent, 100):
+    for batch_it in itertools.batched(recent, 50):
```

Comment:
Whats the reason for the change?

Reply:
For context: This method receives a tuple consisting of IDs and a datetime field that marks the log_cursor. After that, it checks wether the datetime field is older than the older log_cursor, if so, it saves that ID to be used in a batch API endpoint.

The batch API endpoint size limit for the Deals endpoint seems to be limited by 50 by this error i've received when testing locally using a test account:
![Screenshot from 2024-04-16 03-04-00](https://github.com/estuary/connectors/assets/14100959/cf0450a7-6f05-4dd6-90b7-e3bdd6f1065a)
The only streams using this method are Companies, Contacts, Deals and engagements. Never actually stumbled upon this error before, but since they basically use the same endpoint in the V3 API, i figured instead of checking if the Deals streams is being used and impose the limit, already set the limit for all of the 4 streams.
Since it also runs on the batch endpoint and i never had any issues with speed, i figured cutting the size in half didnt really change much performance wise